test/old/testdir/test_options.vim

```diff
@@ -1633,11 +1636,12 @@ func Test_set_boolean_global_local_option()
   call assert_equal(-1, &l:autoread)
   call assert_equal(1, &autoread)
 
-  " :setlocal {option}< removes the local value, so that the global value will be used.
+  " :setlocal {option}< copies the global value to the local value
   setglobal noautoread
   setlocal autoread
   setlocal autoread<
-  call assert_equal(-1, &l:autoread)
+  "call assert_equal(-1, &l:autoread)
+  call assert_equal(0, &l:autoread)
   call assert_equal(0, &autoread)
```

Comment:
Maybe `:set<` and `:setlocal<` should be exchanged for these two tests?

Reply:
I think this is fine as-is. I added some comments, lmk if that's fine

Reply:
I pushed a fixup commit so you see what I mean.